docs/quickstart/docker.md

```diff
@@ -4,7 +4,21 @@ Before you begin, ensure that you have Docker installed on your machine. Docker
 allowing the Roboflow Inference Server to run in a consistent and isolated manner, regardless of the host system. If 
 you haven't installed Docker yet, you can get it from [Docker's official website](https://www.docker.com/get-started).
 
-## Pull
+## via CLI
+
+The easiest way to run the Roboflow Inference Server with Docker is via the command line.
+
+First, [Install the CLI](../index.md#cli).
+
+Running the Inference Server is as simple as running the following command:
+
+```bash
+inference server start
+```
+
+This will pull the appropriate Docker image for your machine and start the Inference Server on port 9001. You can then send requests to the server to get predictions from your model, as described in [HTTP Inference](http_inference.md).
```

Comment:
maybe note the exceptions to the  rule here (trt, any others?)

Reply:
good call, updated!